ParseUI/Classes/Cells/PFCollectionViewCell.m

```diff
@@ -52,6 +52,11 @@ - (void)layoutSubviews {
                                                   CGSizeMake(CGRectGetWidth(bounds), CGRectGetHeight(bounds) - CGRectGetMaxY(imageViewFrame)));
     }
 
+    if (CGRectContainsRect(PFRectMakeWithSize(_imageView.image.size), imageViewFrame)) {
```

Comment:
We need to get the final layout frame, that is affected by _textLabel, but you are right...
I am going to change the whole function flow probably to accommodate for these things.